ide/coqide/fileOps.ml

```diff
@@ -61,35 +61,35 @@ object(self)
         buffer#place_cursor ~where:buffer#start_iter;
         buffer#set_modified false;
         pop_info ();
-        flash_info "Buffer reverted";
+        flash_info "Buffer reloaded";
         Sentence.tag_all buffer;
       with _  ->
         pop_info ();
-        flash_info "Warning: could not revert buffer";
+        flash_info "Warning: could not reload buffer";
     in
     match filename with
       | None -> ()
       | Some f ->
-        if not buffer#modified then do_revert f
+        if not buffer#modified then do_reload f
         else
           let answ = Configwin_ihm.question_box
             ~title:"Modified buffer changed on disk"
-            ~buttons:["Revert from File";
+            ~buttons:["Reload from File";
                       "Overwrite File";
-                      "Disable Auto Revert"]
+                      "Disable Auto Reload"]
```

Comment:
These names in the preferences dialog are lousy, too.  I had to read the code to figure out what they do.  Maybe "Check for modified files" and "Modified file check interval" are better.  "auto" is misleading in that the user has to agree by clicking in the dialog.

![image](https://github.com/coq/coq/assets/1253341/875d0420-f637-447c-bcfd-cfe3aa2fb3e3)